common/lib/mandrill_client/client.py

```diff
@@ -38,40 +38,55 @@ class MandrillClient(object):
     REFERRAL_FOLLOW_UP_EMAIL = 'referred-learner-follow-up'
     REFERRAL_SOCIAL_IMPACT_TOOLKIT = 'social-impact-toolkit'
     MINI_COURSE_ENROLMENT = 'mini-course-enrolment'
+    SEND_ACTION_PLAN = 'send-action-plan'
 
     def __init__(self):
         self.mandrill_client = mandrill.Mandrill(settings.MANDRILL_API_KEY)
 
-    def send_mail(self, template_name, user_email, context, attachments=[], subject=None):
+    def attach_emails(self, user_emails):
+        email_tokens = user_emails.split(',')
+        receiver_emails = []
+        for email in email_tokens:
+            receiver_emails.append({'email': email})
+        return receiver_emails
+
+    def send_mail(self, template_name, user_email, context, attachments=[], subject=None, sender_email=None, images=[]):
         """
         calls the mandrill API for the specific template and email
 
         arguments:
         template_name: the slug/identifier of the mandrill email template
         user_email: the email of the receiver
         context: the data which is passed to the template. must be a dict
+        sender_email:  email for sender
```

Comment:
After changing the param name it would also be changed.

Reply:
Done